packages/frint/index.d.ts

```diff
@@ -1,95 +0,0 @@
-/** Declaration file generated by dts-gen */
```

Comment:
This is not needed, since `tsc` can automatically generate all the exact type definitions.